students/carlos_novoa/lesson09/mailroom.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python3
+import os
+import datetime
+
+now = datetime.datetime.now()
+
+"""
+Lesson9 - Object Oriented Mailroom
+"""
+
+_donors = {
+    'Jim Halpert': [1.00],
+    'Pam Beesley': [1000.00, 2000.00, 3000.00],
+    'Dwight Shrute': [2.00, 3.00],
+    'Michael Scott': [10.00, 20.00, 30.00],
+    'Andy Bernard': [500.00]
+}
+
+
+class Helpers():
+    def format_currency_str(amount=None):
+        return "${0:.2f}".format(float(amount))
```

Comment:
Instance methods automatically pass in the object instance as the first argument, and this is conventionally named 'self', so here `amount` is actually the `Helpers` instance. You can avoid this by making it a static method, but since this is the only method on the class, and there aren't any attributes this should really be a module level function or a method on one of your other classes